heartbeat/docker-compose

```diff
@@ -116,6 +116,8 @@ else
 	COMMAND="$OCF_RESKEY_binpath_default"
 fi
 
+[ "${COMMAND##*/}" = "docker" ] && COMMAND="$COMMAND compose"
```

Comment:
Script defaults to _/usr/bin/docker-compose_.
$COMMAND is then set to _/usr/bin/docker-compose_
If I am correct, this line should be:

```suggestion
[ "${COMMAND##*/}" = "docker-compose" ] && COMMAND="${COMMAND%-compose} compose"
```

Reply:
We also need to be backwards compatible, so can you run `docker --version` and `docker-compose --version` and post the output?

Then I'll be able to use that command to check if we need to add " compose" or not. It would also be helpful if you can run it with v1 of docker compose as well if you have access to nodes with it.

Reply:
I see, backwards compatibility, you are right. Latest versions are;
```
murat@phobos:~$ docker -v ; docker compose version
Docker version 24.0.7, build 24.0.7-0ubuntu4.1
Docker Compose version 2.24.6+ds1-0ubuntu2
```
I will go check my resources if I can install docker compose V1.

Reply:
Docker Compose V1:
`docker-compose version 1.29.2, build unknown`

Reply:
A suggestion: Separate RAs for V1 & V2? Will simplify the script code...

Reply:
That would just be more code to maintain.

Is there one parameter that works for all the versions? Like docker --version, docker compose --version, docker-compose --version?

Reply:
They have done a serious upgrade on docker compose. This is from official documentation: _Unlike Compose V1, Compose V2 integrates into the Docker CLI platform and the recommended command-line syntax is docker compose._

I rechecked; _version_ syntax is strictly different. I could not find a common syntax in between.

How about introducing a new parameter to RA to specify version?